onnxscript/onnx_types.py

```diff
@@ -53,6 +55,13 @@ def __getitem__(self, shape: Optional[ShapeType]):
             shape = (None,)
         return TensorType(self.dtype, shape)
 
+    def __class_getitem__(cls, shape: Optional[ShapeType]):
+        if cls.default_instance is None:
+            raise TypeError(f"{cls} does not specify a default_instance.")
+        # pylint erroneously flags with unsubscriptable-object if
+        # using subscript notation (cls.default_instance[shape]):
+        return cls.default_instance.__getitem__(shape)
```

Comment:
Does this return a type?

Reply:
@abock looks like we still need https://github.com/microsoft/onnx-script/issues/219